gio/ginetaddress.c

```diff
@@ -106,10 +114,18 @@ g_inet_address_set_property (GObject      *object,
       break;
 
     case PROP_BYTES:
+#ifdef HAVE_IPV6
       memcpy (&address->priv->addr, g_value_get_pointer (value),
 	      address->priv->family == AF_INET ?
 	      sizeof (address->priv->addr.ipv4) :
 	      sizeof (address->priv->addr.ipv6));
+#else
+      if (address->priv->family == AF_INET)
+      {
+        memcpy (&address->priv->addr, g_value_get_pointer (value),
+	        sizeof (address->priv->addr.ipv4));
+      }
```

Comment:
- The whole block should be indented by 2 additional spaces.
- The hanging indent should be 4, not 6 spaces.

Reply:
Apologies. It seems that this file (and many others from glib) incorporate both tabs and spaces for their indentation! I assume that frida inherited this from the parent project. So, this looked just fine on my machine!

I replaced the tabs with spaces for the section which I changed. But didn't want to touch anything else as it would make large sections of the project different from the upstream and hence difficult to pull in updates.

Hope that's ok?